cassandra-sink/src/main/java/org/springframework/cloud/stream/module/cassandra/sink/CassandraSink.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.cassandra.sink;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.stream.annotation.EnableBinding;
+import org.springframework.cloud.stream.messaging.Sink;
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHandler;
+
+/**
+ * A sink that can be used to insert data into Cassandra.
+ *
+ * @author Thomas Risberg
+ */
+@EnableBinding(Sink.class)
+public class CassandraSink {
+
+	@Autowired
+	private MessageHandler cassandraSinkMessageHandler;
+
+	@ServiceActivator(inputChannel=Sink.INPUT)
+	public void cassandraSink(Message<?> message) {
+		cassandraSinkMessageHandler.handleMessage(message);
```

Comment:
Looking to this class and how it is wired with the `CassandraSinkConfiguration`, I'd call it as an overhead.
How about rename the `CassandraSinkConfiguration` to this name, move `@EnableBinding(Sink.class)` there, as well as this `@ServiceActivator` annotation?
Otherwise we end up with extra `handleMessage()`...